content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -499,22 +507,35 @@ Common Errors
 There are multiple reasons behind a rejection from the SII, but these are some of the common errors
 you might have and which is the related solution. 
 
-- Error: RECHAZO- DTE Sin Comuna Origen.
+- Error: ``RECHAZO- DTE Sin Comuna Origen.``
   
   *Hint:* Make sure the Company Address is properly filled including the State and City. 
 
-- Error en Monto :  - IVA debe declararse.
+- Error en Monto: ``- IVA debe declararse.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Rut No Autorizado a Firmar.
+- Error: ``Rut No Autorizado a Firmar.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] - 
-  CAF[AAAA-MM-DD]) &gt; 6 meses.
+- Error: ``Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] -
+  CAF[AAAA-MM-DD]) &gt; 6 meses.``
   
   *Hint:* Try to add a new CAF related to this document as the one you’re using is expired.
+  
+- Error: ``Element '{http://www.sii.cl/SiiDte%7DRutReceptor': This element is not expected. Expected
+  is ( {http://www.sii.cl/SiiDte%7DRutEnvia ).``
+  
+  *Hint:* Make sure the field Document Type and VAT are set either in the Customer and in the main
+  company.
+   
+- GLOSA: ``Usuario sin permiso de envio.``
+  
+  *Hint:* This error indicates that most likely, your company has not passed the Certification
+  process in the SII - Sistema de Facturación y Mercadeo. If this is the case, please contact your
```

Comment:
Maybe this could instead be a link to the relevant page e.g.

    `certification process <https://www.sii.cl/factura_electronica/factura_mercado/menu_certificacion.html>`_

or something like that? Or as an external link since there is also a reference to a "Facturación y Mercado" higher in the document which I assume is the same thing despite being spelled out differently?

Also "Mercadeo" might be a typo? Both the SII website and the previous mention use "mercado". (also "de" rather than "y" but I've no idea whether that's relevant).

Reply:
Hi @xmo-odoo the correct word is Mercadeo, so the line that should be corrected is line 57. This documentation will be mainly consumed in spanish actually so I won't recommend to change or translate the name of the system as it is exactly like how it is managed and presented in all the Government documentation: Sistema de Facturación y Mercadeo. Regarding the external Link we can use this one 

 *Hint:* This error indicates that most likely, your company has not passed the `certification process <https://www.sii.cl/factura_electronica/factura_mercado/proceso_certificacion.htm>`_  in the SII - Sistema de Facturación y Mercadeo. If this is the case, please contact your....

@jcs-odoo let me know if it's possible for you to make this changes or if I should close this PR and create a new one with the sugested chanages.  Thank you.

Reply:
@ren-odoo 

> let me know if it's possible for you to make this changes or if I should close this PR and create a new one with the sugested chanages. Thank you.

you should be able to update the files in your branch in a way similar to how you originally created it. Somebody (e.g. jcs or anv) will probably have to squash the commits afterwards but that's not much of an issue.

Reply:
Hi @xmo-odoo thank you for the indications, I just added the changes in an additional commit, by the way you were right about the "Sistema de Facturacion y Mercado" name, I added the text corrections. 

Thank you. 

cc. @jcs-odoo